client/blocks/eligibility-warnings/index.jsx

```diff
@@ -160,4 +162,8 @@ const mapStateToProps = state => {
 	};
 };
 
-export default connect( mapStateToProps )( localize( EligibilityWarnings ) );
+const mapDispatchToProps = {
+	onCancel: () => recordTracksEvent( 'calypso_automated_transfer_eligibility_cancel' )
```

Comment:
Optional:
```jsx
onCancel: partial( recordTracksEvent, 'calypso_automated_transfer_eligibility_cancel' )
```

It'd also be nice to name the handler something more specific like, `recordEligibilityCancel`